spring-xd-yarn/spring-xd-yarn-client/src/main/java/org/springframework/xd/yarn/ClientApplication.java

```diff
@@ -62,7 +63,7 @@ public static void main(String... args) {
 		cmdCommands.add(new YarnClusterDestroyCommand());		
 		app.registerCommands(cmdCommands);
 
-		// shell commands, push not supported in shell
+		// shell commands, push and admininfo not supported in shell
```

Comment:
This didn't work with shell? I had to disable push command because xd-yarn script creates the manifest. Do we have same trouble with this?